src/XdebugHandler.php

```diff
@@ -165,7 +166,7 @@ public static function getAllIniFiles()
      */
     public static function getSkippedVersion()
     {
-        return strval(self::$skipped);
+        return (string) self::$skipped;
```

Comment:
this is up to 6x times faster. I don't think this makes a real difference but there is no reason to use `strval()` instead of casting it directly